be/src/olap/rowset/segment_v2/inverted_index_reader.cpp

```diff
@@ -491,8 +499,15 @@ Status StringTypeInvertedIndexReader::query(OlapReaderStatistics* stats,
                                     result.add(docid);
                                 });
     } catch (const CLuceneError& e) {
-        LOG(WARNING) << "CLuceneError occured, error msg: " << e.what();
-        return Status::Error<ErrorCode::INVERTED_INDEX_CLUCENE_ERROR>();
+        if (_is_range_query(query_type) && e.number() == CL_ERR_TooManyClauses) {
+            LOG(INFO) << "range query term exceeds limits, try to downgrade from inverted index,"
+                      << "column name:" << column_name << " search_str:" << search_str;
+            return Status::Error<ErrorCode::INVERTED_INDEX_FILE_HIT_LIMIT>();
```

Comment:
suggest a more generic name INVERTED_INDEX_IGNORE_INDEX

Reply:
may be INVERTED_INDEX_BYPASS better?